include/matx/transforms/fft.h

```diff
@@ -1065,16 +1061,18 @@ __MATX_INLINE__ void ifft2_impl(OutputTensor o, const InputTensor i,
   params.stream = stream;
   
   // Get cache or new FFT plan if it doesn't exist
-  auto ret = detail::cache_2d.Lookup(params);
-  if (ret == std::nullopt) {
-    auto tmp = new detail::matxFFTPlan2D_t<decltype(in), decltype(out)>{out, in, stream};
-    detail::cache_2d.Insert(params, static_cast<void *>(tmp));
-    tmp->Inverse(out, in, stream);
-  }
-  else {
-    auto fft_type = static_cast<detail::matxFFTPlan2D_t<decltype(in), decltype(out)> *>(ret.value());
-    fft_type->Inverse(out, in, stream);
-  }
+  using cache_val_type = detail::matxFFTPlan2D_t<decltype(out), decltype(in)>;
```

Comment:
did the out/in order get flipped here? versus the old detail::matxFFTPlan2D, the order has changed for the template parameters

Reply:
The previous one was a bug. out, in was correct